lib/chef/util/diff.rb

```diff
@@ -86,6 +83,9 @@ def diff(old_file, new_file)
       # produces a unified-output-format diff with 3 lines of context
       # ChefFS uses udiff() directly
       def udiff(old_file, new_file)
+        require "diff/lcs"
+        require "diff/lcs/hunk"
```

Comment:
that could probably autoload both on Diff::LCS and don't worry about calling out "hunk" specifically

Reply:
As far as I can tell you can only autoload one require per symbol.

Reply:
well that's annoying.

i wonder if we need to write a better autoload implementation or look and see what activesupport has that we might be able to copy....